templates/download/desktop/index.html

```diff
@@ -65,96 +73,153 @@ <h2>Ubuntu {{ releases.latest.full_version }}</h2>
   {% endif %}
 </section>
```

Comment:
You can remove this section and the divider as we only recommend the LTS when its the latest too